artifactory/cli.go

```diff
@@ -999,6 +999,34 @@ func getRetries(c *cli.Context) (retries int, err error) {
 	return retries, nil
 }
 
+func getRetryWaitTime(c *cli.Context) (waitTime int, err error) {
```

Comment:
Let's add a description for this function, specifically saying that the value returned is in milliseconds.